apps/dashboard/src/contract-ui/tabs/tokens/components/burn-button.tsx

```diff
@@ -1,20 +1,44 @@
-import { Icon, useDisclosure } from "@chakra-ui/react";
+import {
+  Sheet,
+  SheetContent,
+  SheetFooter,
+  SheetHeader,
+  SheetTitle,
+  SheetTrigger,
+} from "@/components/ui/sheet";
+import { FormControl, Icon, Input, Stack } from "@chakra-ui/react";
+import { TransactionButton } from "components/buttons/TransactionButton";
+import { useTrack } from "hooks/analytics/useTrack";
+import { useTxNotifications } from "hooks/useTxNotifications";
+import { useState } from "react";
+import { useForm } from "react-hook-form";
 import { FaBurn } from "react-icons/fa";
```

Comment:
whenever u see a `react-icons` import feel free to replace it with an appropriate icon from `lucide-react` please :)